apps/prairielearn/src/components/EditQuestionPointsScore.html.ts

```diff
@@ -94,6 +94,7 @@ function EditQuestionPointsScoreForm({
             class="form-control"
             name="${field}"
             value="${pointsOrScore}"
+            aria-label="${field}"
```

Comment:
This will in some cases name the input `score_perc`, which isn't great. Labels should be human-readable strings. Consider using the technique from `EditQuestionPointsScoreButton` higher up in this file to map the field name to a human-readable label. You could probably reuse the same map between both functions.